### DISCUSSION TITLE
Jetbrains EAP casks

### DISCUSSION THREAD

**Discussion by Moulick** [Homebrew/discussions#684] [Casks]
Would it be a good idea to have casks for Jetbrain IDE EAP versions? There is an EAP version for most JetBrains product

eg: https://www.jetbrains.com/go/nextversion/

**Comment by SMillerDev** [comment]
If you need it you can submit one. I think the `cask-versions` tap would be a good fit.

**Reply by Moulick** [reply]
Sure, I'll be happy too. Just wanted to know there is nothing against it ?

**Comment by dahlia** [comment]
FYI, the upstream homebrew-cask (and even homebrew-cask-versions) decided not to accept JetBrains EAPs and it probably won't be changed in the near future. See also <https://github.com/Homebrew/homebrew-cask/issues/32521>.

Instead, I made a dedicated tap for JetBrains EAPs: [dahlia/jetbrains-eap](https://github.com/dahlia/homebrew-jetbrains-eap).
